server/routes.go

```diff
@@ -70,6 +70,12 @@ func routes(h Handler, conf *config.Config) Routes {
 			Handler: h.GetFromSandbox("stderr"),
 		},
 		Route{
+			Name:    "Kill",
+			Method:  "POST",
```

Comment:
`DELETE` should imho only be used when you're deleting the resource identified by the url. However in this case the task will still be in the database because we want to be able to track that it was killed with later GETs